README.md

```diff
@@ -27,35 +27,41 @@ We welcome feedback, bug reports and bug fixes. We're also happy to discuss feat
 
 Read our [contribution documentation](docs/contributing.md) for additional information on contributing to Javy.
 
-## Build
+## Requirements to build
 
+- On Ubuntu, `sudo apt-get install curl pkg-config libssl-dev clang`
 - [rustup](https://rustup.rs/)
-- Stable Rust (`rustup install stable && rustup default stable`)
+- Stable Rust, installed via `rustup install stable && rustup default stable`
 - wasm32-wasi, can be installed via `rustup target add wasm32-wasi`
 - cmake, depending on your operating system and architecture, it might not be
-  installed by default. On Mac it can be installed with `homebrew` via `brew
-  install cmake`
+  installed by default. On MacOS it can be installed with `homebrew` via `brew
+  install cmake`. On Ubuntu, `sudo apt-get install cmake`.
 - Rosetta 2 if running MacOS on Apple Silicon, can be installed via
   `softwareupdate --install-rosetta`
 
-## Development
+## Development requirements
 
 - wasmtime-cli, can be installed via `cargo install wasmtime-cli` (required for
   `cargo-wasi`)
 - cargo-wasi, can be installed via `cargo install cargo-wasi`
 - wizer, can be installed via `cargo install wizer --all-features`
 - cargo-hack, can be installed via `cargo +stable install cargo-hack --locked`
 
-## Building
+## How to build
 
+Inside the Javy repository, run:
 ```
 $ cargo build -p javy-core --target=wasm32-wasi -r
 $ cargo build -p javy-cli -r
 ```
 
-Alternatively you can run `cargo install --path crates/cli` to install the Javy CLI globally.
+Alternatively you can run `cargo install --path crates/cli` from the root of the Javy repository to install the Javy CLI globally.
```

Comment:
One idea that I realized wasn’t clear before either: The command listed here is an alternative for the second command, not both.

```suggestion
Alternatively you can run `cargo install --path crates/cli` from the root of the Javy repository after building `javy-core` to install the Javy CLI globally.
```

Reply:
Good catch! I opted to go with a slightly different way of phrasing that to try and make the exact instructions and sequence even more explicit.